zeppelin-server/src/test/java/org/apache/zeppelin/rest/NotebookRestApiTest.java

```diff
@@ -120,6 +120,38 @@ public void testGetReloadNote() throws IOException {
       }
     }
   }
+  
+  @Test
+  public void testGetNoteByPath() throws IOException {
```

Comment:
It's better to add a test for querying a not-existed note

Reply:
> It's better to add a test for querying a not-existed note

Thanks @zjffdu  for reviews.  have added test case for querying not-existed note